src/api/app/controllers/webui/projects/public_key_controller.rb

```diff
@@ -4,7 +4,7 @@ class PublicKeyController < WebuiController
       before_action :set_project
 
       def key_dialog
-        render_dialog
+        render_dialog(nil, { project: @project })
```

Comment:
Haml-lint prefer to use local variables  instead of instance variables... and we use that method to fill the content of the dialog, so I need to be able to pass the local variables to the partial.